tv.kodi.Kodi.yml

```diff
@@ -176,8 +176,8 @@ modules:
     buildsystem: cmake-ninja
     sources:
       - type: archive
-        url: "https://github.com/nlohmann/json/archive/refs/tags/v3.10.4.tar.gz"
-        sha256: 1155fd1a83049767360e9a120c43c578145db3204d2b309eba49fbbedd0f4ed3
+        url: "https://github.com/nlohmann/json/archive/refs/tags/v3.11.3.tar.gz"
```

Comment:
I'm unsure, if this dependency is actually needed. So making a note, to check this in a different branch after this lands.